### PR TITLE
Axon Run when a changeset appy fails.

### PR DESCRIPTION
**Pull request by svysali**
Fails Axon Run if any failures are encountered while applying the changeset.

### CODE REVIEW

File: axon.go [hunk 1]
```diff
@@ -260,49 +265,64 @@ func (a *Axon) Shutdown() {
...
+
+	return nil
 }
 
-func (a *Axon) processUpdate(targetDB *sqlx.DB, change *Changeset) {
+func (a *Axon) processUpdate(targetDB *sqlx.DB, change *Changeset) error {
 	pk, err := getPrimaryKeyForChange(change)
 	if err != nil {
 		a.Logger.WithError(err).WithField("table", change.Table).
 			Errorf("unable to process UPDATE for table '%s', changeset has no primary key", change.Table)
```
Comment (bernielomax) [inline axon.go:316]: Not related to your change, but this message needs fixing.

**Review by bernielomax — APPROVED** [review]
Lgtm. Just one comment on something that is not part of this change, but needs correcting.

File: axon.go (hunk 1 above)
Comment (svysali) [reply axon.go:316]: What needs to be fixed here? Print changeset instead of just the table? My brain's not able to figure what's wrong 😫

Comment (bernielomax) [reply axon.go:316]: No the error belongs to get primary key. The message is a dup of the one below it (the actual update)

Comment (svysali) [reply axon.go:316]: It does say `changeset has no primary key`, so I'll leave it as is for now.

**Pull request merged**